packages/visual-editor/src/components/VisualEditorRoot.tsx

```diff
@@ -7,11 +7,27 @@ import { OUTGOING_EVENTS } from '@contentful/experience-builder-core/constants';
 import { useInitializeEditor } from '@/hooks/useInitializeEditor';
 import { useEntityStore } from '@/store/entityStore';
 import { useEditorStore } from '@/store/editor';
+import { useZoneStore } from '@/store/zone';
+import { CTFL_ZONE_ID } from '@/types/constants';
+
+const findNearestDropzone = (element: HTMLElement) => {
```

Comment:
Looking at the branch locally, it looks like this function's return type is being inferred as `any` - should the return type be explicitly added?

```suggestion
const findNearestDropzone = (element: HTMLElement): string | null => {
```